### PR TITLE
[stable/ghost] Add ingress resource. Fix minor bugs

### PR DESCRIPTION
**Pull request by ghost**
**What this PR does / why we need it**:
- Adds ingress based on `helm create`
- Brings helpers into line with `helm create`
- Fix incorrect app version.

### CODE REVIEW

**Comment by ghost** [comment]
cc: @prydonius, @tompizmor, @sameersbn

File: stable/ghost/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: ghost
-version: 3.0.2
-appVersion: 1.22.2
+version: 3.1.0
+appVersion: 1.22.1
```
Comment (carrodher) [inline stable/ghost/Chart.yaml]: Why are you decreasing `appVersion`?

Comment (ghost) [reply stable/ghost/Chart.yaml]: appVersion in Chart.yaml did not match the actual image tag.  This change was to correct that.

Comment (ghost) [reply stable/ghost/Chart.yaml]: Happy to reverse if this is blocking the merge.

Comment (ghost) [reply stable/ghost/Chart.yaml]: @carrodher  ^

Comment (prydonius) [reply stable/ghost/Chart.yaml]: thanks for fixing that @josdotso!

**Review by prydonius — APPROVED** [review]
this is awesome, thanks a lot @josdotso - sorry for the slow review!

/lgtm

**Comment by ghost** [comment]
Thanks!

**Pull request merged**